config/crontab-example

```diff
@@ -37,7 +37,8 @@ MAILTO=!!(*= $mailto *)!!
 48 2 * * * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/user-use-graph
 
 # Once a year :)
-0 0 1 11 * !!(*= $user *)!! /bin/echo "A year has passed, please update the public holidays for the Freedom of Information site, thank you."
+
+0 0 1 11 * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/runner reminder:public_holidays
```

Comment:
Not sure but don'I expect you'll need to be in the app directory for this task as we don't have binstubs installed to be abled to call the bundled rake. Definitely worth testing this in your VM's crontab